docs/notebooks/hugging_face_agent_example.ipynb

```diff
@@ -0,0 +1,878 @@
+    "  <figcaption><stong>Figure 1:</strong> Project ontology.</figcaption>\n",
+    "</figure>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "uzsYd6kPwzFg"
+   },
+   "source": [
+    "[📖 here](https://docs.encord.com/platform-documentation/GettingStarted/gettingstarted-create-ontology) is the documentation for creating ontologies."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "eAJA5_713sNx"
+   },
+   "source": [
+    "# 3.1 Define an Ontology Map\n",
+    "We need to translate the model predictions so that they are paired against the respective Encord ontology item. This is simplest done by using the featureNodeHash of the target"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "hTyFfWb_3g8b"
+   },
+   "outputs": [],
+   "source": [
+    "ontology_map = {\"car\": \"80fUMkkZ\"}"
```

Comment:
The reader has absolutely no chance of pattern matching this ontology hash against the ontology displayed in the image.
I think that we should show either show how to read the ontology hashes via the SDK or describe the "json toggle" in the UI and display it highlighted such that people can follow along.